CMakeLists.txt

```diff
@@ -383,6 +399,13 @@ if(CELERITAS_USE_VecGeom)
       CELERITAS_USE_CUDA
       "${VecGeom_CUDA_FOUND}"
     )
+  elseif(CELERITAS_USE_CUDA
+      AND NOT (VecGeom_CUDA_ARCHITECTURES STREQUAL CMAKE_CUDA_ARCHITECTURES))
```

Comment:
It might be, but who knows how RDC behaves? It would be better to warn if it's at all inconsistent for now.

Reply:
Apriori if the `Celeritas' arch` is a subset of the other architectures then at link time it will bring with only the code for the matching architecture ... (Since to be true on the one example with the finicky `Thurst` kernel) ... but the warning message is lenient enough.